marklogic-langchain4j/src/main/java/com/marklogic/langchain4j/classifier/TextClassifier.java

```diff
@@ -29,8 +37,12 @@ public TextClassifier(String host, String https, String port, String endpoint, S
         Token token;
         try {
             token = tokenFetcher.getAccessToken();
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+            builder = factory.newDocumentBuilder();
         } catch (CloudException e) {
             throw new CloudException("Error retrieving token");
+        } catch (ParserConfigurationException e) {
```

Comment:
Since the constructor declares that it throws `CloudException`, don't catch it, just let it propagate so that we retain the original stacktrace as well (the original error would be lost with this approach).